temp_help/help.yaml

```diff
@@ -0,0 +1,91 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Assuming this will be removed or is it part of a test?

Reply:
This should definitely be removed. It was just a reference help.yaml file.